conf/core/ConfigOptions.asciidoc

```diff
@@ -1855,6 +1880,58 @@ This option allows for passing in a job ID for the currently executing command.
 trying to track multiple commands across a single job. When the Hootenanny API database is used
 for job output storage, the ID will be stored in the job status table of the associated map.
 
+=== josm.map.cleaner.add.detail.tags
```

Comment:
If you're doing feature cleaning but also want an explanation of why a feature was cleaned, turn this on. Obviously, won't work for features that get deleted as a result of cleaning. I thought about putting a list of those in a map tag, but decided to wait on that.